### ISSUE TITLE
bubbles should appear when humans drown

### ISSUE THREAD

**Issue by ATPGaming** [studio-minus/ppg-bugs-and-requests#1824]
**Feature description**
when humans are drowning bubbles should appear

**Feature purpose**
it happens in real life I think and also it makes the game more detailed

**Comment by DindinYT37** [comment]
Isn't this already in the game?

**Comment by DindinYT37** [comment]
> yes but what I meant was bubbles should specifically appear from the human's head. bubbles appear regardless

Yeah, I only remembered bubbles existing and was unsure if this didn't already exist in the game - hence why I asked.

**Comment by RonaBtheCodeR** [comment]
they dont have mouths

**Comment by DindinYT37** [comment]
> they dont have mouths

So?

**Comment by ATPGaming** [comment]
> they dont have mouths

yes they do, have you seen their skeleton?

**Comment by Drogo-man** [comment]
They're like the madness combat people, Thier skeletons clearly have mouths but they lack a face hole.
